test.sh

```diff
@@ -30,6 +30,23 @@ NGINX_LOG=`docker exec container-$STAMP cat /var/log/nginx/error.log`
 printf "\n\nnginx log: \n$NGINX_LOG"
 # TODO: Make assertions against this.
 
+PING_REDIS_OUTSIDE=`docker exec container-$STAMP ping -c 1 container-redis-$STAMP`
+echo; echo # Was getting an error from printf about "\p". Weird.
+echo "ping redis outside:"
+echo $PING_REDIS_OUTSIDE
+
+# TODO
+#PING_REDIS_INSIDE=`docker exec container-$STAMP sh -c '( echo PING | curl -v telnet://container-redis-$STAMP:6379 ) & sleep 1 ; kill $!'`
+#printf "\n\nping redis inside: \n$PING_REDIS_INSIDE"
+# -- OR --
+# `docker exec -it container-$STAMP sh -c "echo 'PING' | nc -w 1 container-redis-$STAMP 6379"`
+# but that requires the installation of netcat
+# -- OR --
+# exec 3<>/dev/tcp/container-redis-$STAMP/6379 && echo -e "PING\n\r" >&3 && cat <&3
+# which requires no installations, but it's crazy, and keeps the connection open.
+# -- OR --
+# install the Redis client
```

Comment:
@pkerpedjiev : I like to test that redis is live, and not just that the host is up. I would probably lean towards the netcat option, but if you think the redis command line tool would have other uses we could do that. It's another apt-get, invalidating most of the layers, so I didn't want to do anything rash.